stix_shifter_utils/modules/base/stix_transmission/base_connector.py

```diff
@@ -89,14 +89,23 @@ def ping_connection(self):
         """
         raise NotImplementedError()
 
-    def create_results_stix_connection(self, entry_point, search_id, offset, length, data_source):
+    def create_results_stix_connection(self, entry_point, search_id, offset, length, data_source, metadata=None):
         stats = []
-        result = entry_point.create_results_connection(search_id, offset, length)
+        if metadata:
+            result = entry_point.create_results_connection(search_id, offset, length, metadata)
+        else:
+            result = entry_point.create_results_connection(search_id, offset, length)
         stats.append({'action': 'transmission', 'time': int(time.time()*1000)})
+        metadata = None
+        if 'metadata' in result:            
+            metadata = result['metadata']
+            del result['metadata']
```

Comment:
just wondering, is it just to avoid passing metadata in results translation?

Reply:
yes, no need for metadata in translation.